src/views/SemesterAdminTable.vue

```diff
@@ -0,0 +1,374 @@
+<template>
```

Comment:
I switched to using the built-in bootstrap-vue table component for this table, instead of using the bootstrap-table library that we had been using before.

Reply:
good call!